web/api/v1/errors_test.go

```diff
@@ -253,6 +253,6 @@ func (DummyRulesRetriever) RuleGroups() []*rules.Group {
 }
 
 // AlertingRules implements RulesRetriever.
-func (DummyRulesRetriever) AlertingRules() []*rules.AlertingRule {
+func (DummyRulesRetriever) AlertingRules(matcherSets ...[]*labels.Matcher) []*rules.AlertingRule {
```

Comment:
```suggestion
func (DummyRulesRetriever) AlertingRules(_ ...[]*labels.Matcher) []*rules.AlertingRule {
```

Reply:
remove this parameter as we are not use this parameter at alerts api